op-e2e/actions/proofs/isthmus_setcode_tx_test.go

```diff
@@ -189,3 +196,224 @@ func testInvalidSetCodeTxBatch(gt *testing.T, testCfg *helpers.TestCfg[any]) {
 
 	env.RunFaultProofProgram(t, l2safe.Number, testCfg.CheckResult, testCfg.InputParams...)
 }
+
+func Test_ProgramAction_SetCodeTxWithContractCreationBitSet(gt *testing.T) {
+	matrix := helpers.NewMatrix[any]()
+	defer matrix.Run(gt)
+
+	matrix.AddDefaultTestCases(
+		nil,
+		helpers.LatestForkOnly,
+		runSetCodeTxTypeWithContractCreationBitSetTest,
+	)
```

Comment:
```suggestion
	matrix.AddDefaultTestCases(
		nil,
		helpers.LatestForkOnly,
		runSetCodeTxTypeWithContractCreationBitSetTest,
	)
	matrix.Run(gt)
```
I don't know who came up with the idea to _not_ simply call run at the end...